### PR TITLE
Android build, configuration, and login support

### PR DESCRIPTION
**Pull request by arthur740212**
Main features added on this branch :
Android configuration editor, save/loading config files
Logging with Account Portal and Persistent Login

Fixes:
Changed Android/Editor to AndroidEditor/Editor due to Unity predefined file names for Android

Ongoing : Outlines for RTC support

### CODE REVIEW

File: Assets/Plugins/AndroidEditor/Editor/PlatformSpecificConfigEditorAndroid.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+            EpicOnlineServicesConfigEditor.AssigningFlagTextField("Override Platform Flags (Seperated by '|')", 190, ref overrideValues.platformOptionsFlags);
+
+            EpicOnlineServicesConfigEditor.AssigningFloatToStringField("Override initial button delay for overlay", 250, ref overrideValues.initialButtonDelayForOverlay);
+
+            EpicOnlineServicesConfigEditor.AssigningFloatToStringField("Override repeat button delay for overlay", 250, ref overrideValues.repeatButtonDelayForOverlay);
+
+            EpicOnlineServicesConfigEditor.AssigningBoolField("HACK: force send input without delay", 250, ref overrideValues.hackForceSendInputDirectlyToSDK);
+
+            //EpicOnlineServicesConfigEditor.AssigningBoolField("Use Eased Restriction NRO path", 250, ref configFile.currentEOSConfig.useEasedRestrictionNROPath);
```
Comment (andrew-hirata-playeveryware) [inline Assets/Plugins/AndroidEditor/Editor/PlatformSpecificConfigEditorAndroid.cs]: This has be deleted

File: Assets/Plugins/AndroidEditor/Editor/PlatformSpecificConfigEditorAndroid.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                overrideValues = configFile.currentEOSConfig.overrideValues;
+            }
+
+            EpicOnlineServicesConfigEditor.AssigningFlagTextField("Override Platform Flags (Seperated by '|')", 190, ref overrideValues.platformOptionsFlags);
+
+            EpicOnlineServicesConfigEditor.AssigningFloatToStringField("Override initial button delay for overlay", 250, ref overrideValues.initialButtonDelayForOverlay);
+
+            EpicOnlineServicesConfigEditor.AssigningFloatToStringField("Override repeat button delay for overlay", 250, ref overrideValues.repeatButtonDelayForOverlay);
+
+            EpicOnlineServicesConfigEditor.AssigningBoolField("HACK: force send input without delay", 250, ref overrideValues.hackForceSendInputDirectlyToSDK);
```
Comment (andrew-hirata-playeveryware) [inline Assets/Plugins/AndroidEditor/Editor/PlatformSpecificConfigEditorAndroid.cs]: This config option can be removed

File: Assets/Scripts/EOSLobbyManager.cs [hunk 3]
```diff
@@ -2185,7 +2187,7 @@ private void AddLocalUserAttributes()
         /// <param name="JoinLobbyCompleted">Callback when join lobby is completed</param>
         public void JoinLobby(string lobbyId, LobbyDetails lobbyDetails, bool presenceEnabled, OnLobbyCallback JoinLobbyCompleted)
         {
-
+
```
Comment (andrew-hirata-playeveryware) [inline Assets/Scripts/EOSLobbyManager.cs]: This is a pedantic request, but could you delete the whitespace that was added here?

**Review by andrew-hirata-playeveryware — CHANGES_REQUESTED** [review]

File: Assets/Scripts/EOSLobbyManager.cs (hunk 3 above)
Comment (arthur740212) [reply Assets/Scripts/EOSLobbyManager.cs]: Cleaned up

File: Assets/Plugins/AndroidEditor/Editor/PlatformSpecificConfigEditorAndroid.cs (hunk 1 above)
Comment (arthur740212) [reply Assets/Plugins/AndroidEditor/Editor/PlatformSpecificConfigEditorAndroid.cs]: Cleaned up

File: Assets/Plugins/AndroidEditor/Editor/PlatformSpecificConfigEditorAndroid.cs (hunk 2 above)
Comment (arthur740212) [reply Assets/Plugins/AndroidEditor/Editor/PlatformSpecificConfigEditorAndroid.cs]: Cleaned up

**Pull request merged**